homeworks/Yaroslav.Belinskyi_YaroslavBelinskyi/homework_5/firstscenario.js

```diff
@@ -0,0 +1,68 @@
+// 1 task
+const exampleArray = [10, -10, 10, -10, 10];
+const resultArray = [];
+function sumArray(arr) {
+    arr.reduce((previousSum, currentNumber) => {
+        const value = previousSum + currentNumber;
+        resultArray.push(value);
```

Comment:
pushing to array in global scope? not nice at all...

Reply:
fixed